infrastructure/cdn-in-a-box/README.md

```diff
@@ -22,17 +22,12 @@ This is intended to simplify the process of creating a "CDN in a box", easing
 the barrier to entry for newcomers as well as providing a way to spin up a
 minimal CDN for full system testing.
 
-## Implemented Components
-As of the time of this writing, Traffic Ops, Traffic Monitor, Traffic Portal and a
-database server for Traffic Ops are all fully implemented. An edge-tier cache,
-mid-tier cache and simple origin are also all implemented, although with limited
-functionality (caches do not respond to update queues/routing, and origin serves
-static HTTP content - likely subject to change in favor of proper video streaming in
-the future). Other components will follow as well as details on specific parts of the
-implementation.
+> **note**
+>
+> For a more in-depth discussion of the CDN in a Box system, please see [the official documentation](https://traffic-control-cdn.readthedocs.io/en/latest/admin/quick_howto/ciab.html).
```

Comment:
It can't be relative to reStructuredText from markdown - markdown is self-contained and has no notion of other files. 
However, github provides extensions to markdown that give the ability to reference a file within the same repository - but I don't think that's a good idea because it can only reference the documentation source, not the built, ready-to-be-read Sphinx output.
As far as forward-looking links, the best I can do is that `/latest/` in the URL

The other option is to switch the README to be reStructuredText instead of Markdown, but I'm hesitant to do that because although it would place the documentation for the stack in one place and would allow for it to use relative references, it would mean that the links would necessarily be broken - and possibly not render properly - when viewing the README itself, outside of the documentation.

Reply:
fair enough..